core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -1181,7 +1181,7 @@ static function ($allowedTag)
 								foreach ($GLOBALS['TL_HOOKS']['insertTagFlags'] as $callback)
 								{
 									$this->import($callback[0]);
-									$varValue = $this->{$callback[0]}->{$callback[1]}($flag, $tag, $arrCache[$strTag], $flags, $blnCache, $tags, $arrCache, $_rit, $_cnt); // see #5806
+									$varValue = $this->{$callback[0]}->{$callback[1]}($flag, $tag, $arrCache[$strTag], $flags, $blnCache, $tags, array(), $_rit, $_cnt); // see #5806
```

Comment:
Here `$arrCache[$strTag]` is not about some cache but actually the current value of the insert-tag.